### PR TITLE
Color design token import

### PR DESCRIPTION
**Pull request by bradfrost**
### Summary:
Shortcut link: https://app.shortcut.com/czi-edu/story/189626/import-color-tokens-into-json
Figma file: https://www.figma.com/file/IfMWAtWqNrQprImlco4pi0/SummitLearning.com-Foundations-(DRAFT)?node-id=0%3A1

This PR imports the color tokens structure and values from [the Figma file](https://www.figma.com/file/IfMWAtWqNrQprImlco4pi0/SummitLearning.com-Foundations-(DRAFT)?node-id=0%3A1), replacing the existing token architecture and naming.

A few notes: 
- This only touches color tokens
- Typography tokens are outstanding
- This PR doesn't update the individual component file CSS custom properties (which will be tackled by @ifrost1 and @tpitre-czi )

Because of those outstandings, I'm leaving this as a draft until we get typography tokens represented and new token CSS custom properties wired up to the components.

### CODE REVIEW

**Comment by anniehu4** [comment]
Thanks! Added Diedra & Jin for awareness since this seems related to some of the questions we've had

**Review by tpitre — APPROVED** [review]
This all looks good. I'm going to merge this branch into our temporary `next-big-medium` working branch and branch off of that to continue with the typography token presets.

**Comment by bradfrost** [comment]
@anniehu4 @tpitre-czi This PR introduces the new color tokens into the mix. It does _not_ cover typography tokens nor does it wire up the tokens to the components. Annie, how would you like to handle this? I think there's two ways: 
1. We can merge this into `next`, quickly merge in TJ's typography token work, and then update the component files accordingly. 
2. We can create a new `tokens` branch to handle all of the token work, and only when that's done merge into `next`.

I have a preference for approach 1 because it gets things into place sooner rather than later, even if there will be some broken components along the way. But I'll defer to you Annie on how you want to handle this!

**Comment by bradfrost** [comment]
@anniehu4 Per our conversation, changing this to a real PR from a draft in order to get the color tokens into `next`.

**Review by anniehu4 — APPROVED** [review]

File: src/design-tokens/css/base/body.css [hunk 1]
```diff
@@ -4,7 +4,7 @@ body {
   @mixin eds-theme-typography-body-text;
   display: flex;
   flex-direction: column;
-  color: var(--eds-theme-color-body-foreground);
+  color: var(--eds-theme-color-neutral-default);
```
[the remark this answers is not in the document]
Comment (bradfrost) [reply src/design-tokens/css/base/body.css]: @anniehu4 it should! Just updated.

**Pull request merged**